controllers/articles.controller.js

```diff
@@ -20,3 +20,18 @@ exports.getArticles = (req, res, next) => {
         })
         .catch(next);
 };
+
+exports.patchArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (inc_votes === undefined) {
+      return res.status(400).send({ msg: 'Missing inc_votes' });
+  }
```

Comment:
Error handling like this should be done in the model instead of here in the controller